lms/envs/devstack.py

```diff
@@ -215,6 +215,8 @@ def should_show_debug_toolbar(_):
 CORS_ORIGIN_WHITELIST = ()
 CORS_ORIGIN_ALLOW_ALL = True
 
+################################ CCX ###############################
+FEATURES['CUSTOM_COURSES_EDX'] = True
```

Comment:
Question: was this change made just for your local testing, or are you planning to merge this and now enable this on devstack?